src/Tizen.Sensor/Tizen.Sensor/Sensor.cs

```diff
@@ -24,12 +25,60 @@ internal static class Globals
         internal const string LogTag = "Tizen.Sensor";
     }
 
+    /// <summary>
+    /// Data from sensor
+    /// </summary>
+    [StructLayout(LayoutKind.Sequential, Pack = 0)]
+    public struct SensorEventStruct
```

Comment:
Do not directly expose native data struct

Reply:
There is a public API "ProviderPublish" which has this "SensorEventStruct" as  parameter. So we created a public struct.